scripts/json-to-env.sh

```diff
@@ -118,6 +118,10 @@ jq -r  '
         {
             "path": "MAX_CSV_FILE_SIZE",
             "env_var": "MAX_CSV_FILE_SIZE"
+        },
+        {
+            "path": "SERVICE_MANAGEMENT_REFERENCE",
```

Comment:
This doesn't need to be mapped at output as it isn't used

Reply:
fixed